components/builder-jobsrv/src/server/scheduler.rs

```diff
@@ -404,6 +404,9 @@ impl ScheduleMgr {
         {
             // Check the deps for the project. If we don't find any dep that
             // is in our project list and needs to be built, we can dispatch the project.
+            // NOTE: get_ident().is_empty() is only true if the project has never been built
+            // Otherwise ident is going to be the FQPI of the latest ident (which channel is
+            // unclear)
```

Comment:
Might be worth enshrining this in code directly, rather than just in a comment.

Reply:
I didn't want to make any changes in the old scheduler code, but I did want to note this to clarify something I found confusing.